modules/command/dh/dh-pay-out-command.js

```diff
@@ -154,8 +154,14 @@ class DhPayOutCommand extends Command {
         const profile = await this.blockchain
             .getProfile(blockchainIdentity, blockchain_id).response;
 
-        const walletBalance =
-            await this.blockchain.getWalletTokenBalance(node_wallet, blockchain_id).response;
+        let walletBalance;
+        if (blockchain_title !== constants.BLOCKCHAIN_TITLE.Starfleet) {
```

Comment:
Let's avoid using keyword Starfleet, it is deprecated. I suggest parachain or similar

Reply:
Parachain is kind of too generic, do you have a recommendation what we would use in this case?
Also, the repo is called starfleet parachain, i don't understand how it's deprecated?

Reply:
Due to copyright policy, we are no longer able to use Starfleet keyword. On the polkadotjs apps it is listed as OriginTrail Parachain so I suggest to use "origintrail-parachain" or similar. Also, we should rename the repository soon.